src/features/canvass/utils/getVisitPercentage.ts

```diff
@@ -0,0 +1,68 @@
+import { Household } from 'features/areaAssignments/types';
+
+export interface VisitStats {
+  totalSuccessfulVisits: number;
+  totalVisits: number;
+}
+
+export function getVisitPercentage(
```

Comment:
This calculation will be made much easier with the real backend, and then we can probably remove this function altogether. So I haven't reviewed it very thoroughly for now.